LICENSE

```diff
@@ -175,3 +175,10 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+
+This product contains code from OCI/opencontainers, which is available
+under an Apache License 2.0. For details, see container/statistics.go, container/cgroupfs/blkio.go.
+
+This product contains code from Google cAdvisor, which is available
+uder an Apache License 2.0. For details, see container/fs/fs.go, container/fs/types.go, container/network/network.go, container/network/tcp.go
```

Comment:
In mentioned files (container/statistics.go, container/cgroupfs/blkio.go, ontainer/fs/fs.go, container/fs/types.go, container/network/network.go, container/network/tcp.go) if there are original to Intel but with snippets from 3rd party components - files should have two Copyrights headers: Intel + 3rd component, see [this](https://github.com/intelsdi-x/snap-plugin-publisher-heapster/blob/master/exchange/exchange.go) example.